src/Parser/DeathKnight/Blood/Modules/Features/MarrowrendUsage.js

```diff
@@ -29,10 +34,18 @@ class MarrowrendUsage extends Analyzer {
   refreshMRCasts = 0;
   totalMRCasts = 0;
 
-  REFRESH_AT_STACKS = 6;
-  REFRESH_AT_SECONDS = 6;
-  BS_DURATION = 30;
-  MR_GAIN = 3;
+  hasBonesOfTheDamned = false;
+
+  bonesOfTheDamnedProc = 0;
+  totalStacksGenerated = 0;
+
+  constructor(...args) {
+    super(...args);
+
+    if(this.selectedCombatant.hasTrait(SPELLS.BONES_OF_THE_DAMNED.id)) {
```

Comment:
You can skip the if statement and just set `this.hasBonesOfTheDamned to the value it returns instead